executor/aggregate.go

```diff
@@ -501,6 +502,10 @@ func (w *HashAggFinalWorker) run(ctx sessionctx.Context, waitGroup *sync.WaitGro
 
 // Next implements the Executor Next interface.
 func (e *HashAggExec) Next(ctx context.Context, chk *chunk.Chunk) error {
+	if e.runtimeStat != nil {
+		start := time.Now()
+		defer e.runtimeStat.Record(time.Now().Sub(start), chk.NumRows())
```

Comment:
Defer function input parameter will be calculated in defer define.
```go
func main() {
        foo()
        foo1()
}

func foo() {
        start := time.Now()
        defer func() {
                fmt.Println(time.Since(start))
        }()
        time.Sleep(1 * time.Second)
}

func foo1() {
        start := time.Now()
        defer fmt.Println(time.Since(start))
        time.Sleep(1 * time.Second)
}
```
output:
```shell
1.001687191s
319ns
```

Reply:
emmm.....that's right..@zz-jason  u mislead me.:rofl:

Reply:
😂I'm sorry